### ISSUE TITLE
Search/filter and sort transactions

### ISSUE THREAD

**Issue by curtisblackwell** [TinyAnvil/tiny-money-webpack#6]
Looking through history for what appears to be a duplicate (possibly a 🐛) is difficult with so many transactions and no way to limit the number shown.

**Comment by kalepail** [comment]
Definitely a feature I want. Nothing super crazy but sorting by amount / date as well as searching a date range are things I'd definitely like to add asap.

**Comment by curtisblackwell** [comment]
Agreed. I'm thinking something as simple as the [grid component example](https://vuejs.org/v2/examples/grid-component.html). I have a component I based off of that for LMD that might work well, but we haven't written docs for it yet.

I think a date range filter makes sense as a separate feature, though. I'd love to see the date range component you built for Baremetrics sans jQuery and more easily styled.

**Comment by kalepail** [comment]
Both great ideas. I've been wanting to rebuild the BM Calendar as a Vue Component for awhile. Won't have time for that for awhile but it's definitely something I'm hoping to get to eventually.
